tests/measurements/test_purity_measurement.py

```diff
@@ -293,7 +293,7 @@ def circuit(x):
         assert qml.math.allclose(purity, expected_purity)
 
     @pytest.mark.jax
-    @pytest.mark.parametrize("device", grad_supported_devices)
+    @pytest.mark.parametrize("device", ["default.qubit"])
```

Comment:
Oh this one is because this testclass used to fail at several mixed-specific points which I xfailed later. It's been a while so I need to double check if their bugs already resolved and this whole modification can be simply reverted.

Reply:
I confirm that the whole change here can be reverted.